config/locales/en.yml

```diff
@@ -814,6 +814,12 @@ en:
     system_checks:
       database_schema_check:
         message_html: There are pending database migrations. Please run them to ensure the application behaves as expected
+      elasticsearch_health_red:
+        message_html: Elasticsearch cluster is unhealthy (red status), search features are unavailable
+      elasticsearch_health_yellow:
+        message_html: Elasticsearch cluster is unhealthy (yellow status), search features may be unavailable
+      elasticsearch_index_mismatch:
+        message_html: Elasticsearch indexes do not match current definitions. Please run <code>tootctl search deploy --only=%{value}</code>
```

Comment:
Might be more helpful to say they are outdated? I'm thinking if indexes, index mappings, or index schemas would be best to not lose information in the sentence (to differentiate from *data* being outdated)

Reply:
Right, I did not want to say they were “outdated” as you could be downgrading from e.g. a feature branch, but honestly this nuance is probably not needed.

“Elasticsearch indexes are outdated. Please run …” does sound more understandable and will be as accurate in almost all cases.